src/Authentication.Abstractions/Authentication.Abstractions.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <IncludeHyak>true</IncludeHyak>
+    <IsCmdletProject>true</IsCmdletProject>
```

Comment:
This is to bring in the dependency of PowerShell SDK because config API needs `InvocationInfo`